compass/landice/tests/humboldt/mesh.py

```diff
@@ -0,0 +1,361 @@
+        self.add_input_file(filename='Humboldt.geojson',
+                            target='Humboldt.geojson',
+                            database=None, copy=True)
+        self.add_input_file(filename='greenland_8km_2020_04_20.epsg3413.nc',
+                            target='greenland_8km_2020_04_20.epsg3413.nc',
+                            database='')
+
+    # no setup() method is needed
+
+    def run(self):
+        """
+        Run this step of the test case
+        """
+        logger = self.logger
+        try:
+            config = self.config
+            section = config['humboldt']
+        except NameError:
+            print('No config file; skipping')
+
+        print('calling build_cell_wdith')
+        cell_width, x1, y1, geom_points, geom_edges = self.build_cell_width()
+        print('calling build_planar_mesh')
+        build_planar_mesh(cell_width, x1, y1, geom_points,
+                          geom_edges, logger=logger)
+        dsMesh = xarray.open_dataset('base_mesh.nc')
+        print('culling mesh')
+        dsMesh = cull(dsMesh, logger=logger)
+        print('converting to MPAS mesh')
+        dsMesh = convert(dsMesh, logger=logger)
+        print('writing grid_converted.nc')
+        write_netcdf(dsMesh, 'grid_converted.nc')
+        # If no number of levels specified in config file, use 10
+        try:
+            levels = section.get('levels')
+        except NameError:
+            levels = '10'
+        print('calling create_landice_grid_from_generic_MPAS_grid.py')
+        args = ['create_landice_grid_from_generic_MPAS_grid.py',
+                '-i', 'grid_converted.nc',
+                '-o', 'gis_1km_preCull.nc',
+                '-l', levels, '-v', 'glimmer']
+        check_call(args, logger=logger)
+
+        # This step uses a subset of the whole Greenland dataset trimmed to
+        # the region around Humboldt Glacier, to speed up interpolation.
+        # This could also be replaced with the full Greenland Ice Sheet
+        # dataset.
+        print('calling interpolate_to_mpasli_grid.py')
+        args = ['interpolate_to_mpasli_grid.py', '-s',
+                'humboldt_1km_2020_04_20.epsg3413.icesheetonly.nc', '-d',
+                'gis_1km_preCull.nc', '-m', 'b', '-t']
+
+        check_call(args, logger=logger)
+
+        # This step is only necessary if you wish to cull a certain
+        # distance from the ice margin, within the bounds defined by
+        # the GeoJSON file.
+        print('calling define_cullMask.py')
+        args = ['define_cullMask.py', '-f',
+               'gis_1km_preCull.nc', '-m'
+               'distance', '-d', '5.0']
+
+        check_call(args, logger=logger)
+
+        # This step is only necessary because the GeoJSON region
+        # is defined by lat-lon.
+        print('calling set_lat_lon_fields_in_planar_grid.py')
+        args = ['set_lat_lon_fields_in_planar_grid.py', '-f',
+                'gis_1km_preCull.nc', '-p', 'gis-gimp']
```

Comment:
We need @hollyhan's fix in https://github.com/MPAS-Dev/MPAS-Tools/pull/450 for the longitude variables to be in the desired range.  I don't know if this causes issues with `MpasMaskCreator.x`.  Presumably you would have noticed if it did.